Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN \
   apk --update --upgrade \
     add ca-certificates --virtual .buildDeps git ca-certificates openssl g++ make python3 linux-headers \
   && update-ca-certificates \
-  && corepack install -g pnpm@9 \
+  && corepack install -g pnpm@9.15.4 \
```

Comment:
remove this all-together, as well as COREPACK_ENABLE_NETWORK=1
use recent node 22.13.1+ - it will include pnpm 9+